### ISSUE TITLE
Resize and align images

### ISSUE THREAD

**Issue by BjoernRave** [nhn/tui.editor#861]
I would like to be able to resize and align images

**Comment by seonim-ryu** [comment]
@BjoernRave Actually, this explanation is not enough to know exactly what you want. Whether you want to handle image size or alignment in a layer popup, or whether you want to add markdown syntax, I need a detailed explanation.

First of all, as I'm expecting, you can use html in the Markdown editor, and use the style attribute to handle image size and alignment. Did you want this feature?
```md
<img src="..." style="width:50%;height:50%;float:right;" />
```
![mde](https://user-images.githubusercontent.com/18183560/78222491-4ec1e080-7500-11ea-8964-1f6060de645d.png)

**Comment by BjoernRave** [comment]
yes, I would like to be able, that when I add an image to the wysiwyg editor, that I can resize and align the image inside the editor

**Comment by g-rodigy** [comment]
+1
Resize and align the image inside the editor or popup

**Comment by KamaMakh** [comment]
+1
Resize and align the image inside the editor or popup, maybe add a margin

**Comment by jdtrat** [comment]
Would love to have this for the WYSIWYG editor! For the markdown side, perhaps a macro would work? Something like that used in [remark.js](https://github.com/gnab/remark/issues/72#issuecomment-62225566) may be helpful. From the linked issue:

```js
remark.macros.scale = function (percentage) {
  var url = this;
  return '<img src="' + url + '" style="width: ' + percentage + '" />';
};
```

Usage:

`
![:scale 50%](image.jpg)
`

Output:

```html
<img src="image.jpg" style="width: 50%" />
```

**Comment by Teolone88** [comment]
+1
Resize and align the image inside the editor or popup, maybe add a margin

**Comment by livevsonline** [comment]
Has anyone found a workaround for now for this problem?

**Comment by ycshill** [comment]
同问，希望能解决一下这个问题啊！
